sdk/python/feast/infra/registry/base_registry.py

```diff
@@ -248,14 +248,16 @@ def get_stream_feature_view(
 
     @abstractmethod
     def list_stream_feature_views(
-        self, project: str, allow_cache: bool = False
+        self, project: str, allow_cache: bool = False, feast_apply_operation: bool = False,
```

Comment:
Nice! 

Nit: with respect to the parameter name, we might want to ignore udfs outside of calling feast_apply, so it might help to give it a more general name.